docs/topics/mta-web-creating-migration-waves.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-migration-waves_{context}"]
+= Creating migration waves
+
+A migration wave is a tool for creating Jira issues for a group of applications that are to be migrated at the same time. For each application assigned to a migration wave, {ProductShortName} opens a separate Jira issue with an automatically generated title.
+
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. Click *Create new*.
++
+The *New migration wave* window opens.
+. Enter the following information or select from drop-down lists:
+* Name
```

Comment:
Name is optional (in which case the start and end dates are used to distinguish migration waves.

Reply:
Corrected